autogen/agentchat/contrib/vectordb/base.py

```diff
@@ -207,6 +207,10 @@ def create_vector_db(db_type: str, **kwargs) -> VectorDB:
             from .pgvectordb import PGVectorDB
 
             return PGVectorDB(**kwargs)
+        if db_type.lower() in ["mdb", "mongodb", "atlas"]:
+            from .mongodb import MongoDBAtlasVectorDB
+
+            return MongoDBAtlasVectorDB(**kwargs)
```

Comment:
can you add the tests for the `mongodb` at [test dir](https://github.com/microsoft/autogen/tree/main/test/agentchat/contrib/vectordb). thanks

Reply:
Added `test_mongodb.py`